CONTRIBUTING.md

```diff
@@ -22,3 +22,39 @@ To navigate our codebase with confidence, see [Pynecone Docs](https://pynecone.i
 
 Thank you for supporting Pynecone!🎊
 
+## 💻 How to Run a Local Build of Pynecone 
+Here is a quick guide to how the run Pynecone repo locally so you can start contributing to the project.
+
+First clone Pynecone:
+```
+git clone https://github.com/pynecone-io/pynecone.git
+```
+
+Navigate into the repo:
+```
+cd pynecone
+```
+
+Now create an examples folder so you can test the local Python build in this repository:
+```
+mkdir examples
+cd example
```

Comment:
You mkdir examples and then cd into example, should be plural

Reply:
Thanks for point this out fixing now